custom/enikshay/private_sector_datamigration/migrations/0006_auto_20170520_1638.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-05-20 16:38
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('private_sector_datamigration', '0005_auto_20170520_1530'),
```

Comment:
Can't you just put both migrations in this file and set this to `('private_sector_datamigration', '0004_migratedbeneficiarycounter'),`?

I don't think the migration framework requires that every migration file is present as long as the most recent one is there and its dependencies are set correctly.